### PR TITLE
integrate i18n react

### PR DESCRIPTION
**Pull request by zzacharo**

### CODE REVIEW

File: invenio_app_rdm/theme/translations/invenio_app_rdm/i18next-scanner.config.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 Graz University of Technology.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
```
Comment (mb-wali) [inline invenio_app_rdm/theme/translations/invenio_app_rdm/i18next-scanner.config.js:4]:
```suggestion
// Invenio App RDM is free software; you can redistribute it and/or modify it
```

File: invenio_app_rdm/theme/translations/invenio_app_rdm/i18next.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 Graz University of Technology.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
```
Comment (mb-wali) [inline invenio_app_rdm/theme/translations/invenio_app_rdm/i18next.js:4]:
```suggestion
// Invenio App RDM is free software; you can redistribute it and/or modify it
```

File: invenio_app_rdm/theme/translations/invenio_app_rdm/i18next.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  },
+};
+// i18next instance creation
+// https://www.i18next.com/overview/api#instance-creation
+// this is required in order to keep the resources seperate
+// if there is going to be another package
+// which requires translation this is the way to create a new instance.
+//
+// We can use this particular instance for this particular package
+// to mark strings for translation.
```
Comment (mb-wali) [inline invenio_app_rdm/theme/translations/invenio_app_rdm/i18next.js:33]: i think this is now clear, should get rid of.
```suggestion

```

File: invenio_app_rdm/theme/translations/invenio_app_rdm/scripts/initCatalog.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 Graz University of Technology.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
```
Comment (mb-wali) [inline invenio_app_rdm/theme/translations/invenio_app_rdm/scripts/initCatalog.js:4]:
```suggestion
// Invenio App RDM is free software; you can redistribute it and/or modify it
```

**Review by mb-wali — COMMENTED** [review]
LGTM! you properly need to run `check-manifest` to add the jsons.

**Comment by lnielsen** [comment]
Looks fine to me. Yes, probably we can do something smarter in the future, so that it works more easily for other modules as well, but I'm ok with merging it now. 

@ntarocco Can you have a quick look if you see anything?

**Review by ntarocco — APPROVED** [review]

**Pull request merged**